torch/_inductor/codegen/wrapper.py

```diff
@@ -1063,6 +1073,47 @@ def codegen_model_constructor(self):
                         f"inputs_info_[{idx}].shape.emplace_back({size}, {size}, nullptr);"
                     )
 
+            if V.graph.aot_mode:
+                curr_offset = 0
+                for idx, (name, tensor) in enumerate(V.graph.constants.items()):
+                    assert isinstance(tensor, torch.Tensor)
+                    self.prefix.writeline(
+                        f"""constants_info_[{idx}].name = "{name}";"""
+                    )
+                    self.prefix.writeline(
+                        f"constants_info_[{idx}].dtype = {DTYPE_TO_ATEN[tensor.dtype]};"
+                    )
+                    tensor = tensor.contiguous()
+                    data_size = tensor.nelement() * tensor.element_size()
+                    self.prefix.writeline(
+                        f"constants_info_[{idx}].offset = {curr_offset};"
+                    )
+                    curr_offset += data_size
+                    self.prefix.writeline(
+                        f"constants_info_[{idx}].data_size = {data_size};"
+                    )
+                    if data_size == 0:
+                        V.graph.aot_constants.append(b"")
```

Comment:
Sorry maybe I don't understand the code too well here, but why are we appending to `V.graph.aot_constants` here as opposed to in graph.py, like where [`constants_repr` is updated](https://www.internalfb.com/code/fbsource/[a0c5bafcce08]/fbcode/caffe2/torch/_inductor/graph.py?lines=511)? It feels weird to be modifying `V.graph` in this file, which is just codegen-ing the wrapper.